include/crm/pengine/common.h

```diff
@@ -105,17 +105,17 @@ enum rsc_role_e {
 #  define RSC_ROLE_MASTER_S  "Master"
 
 enum pe_print_options {
-    pe_print_log            = (1 << 0),
-    pe_print_html           = (1 << 1),
-    pe_print_ncurses        = (1 << 2),
-    pe_print_printf         = (1 << 3),
+    pe_print_log            = (1 << 0),  //! \deprecated Use output messages instead
+    pe_print_html           = (1 << 1),  //! \deprecated Use output messages instead
+    pe_print_ncurses        = (1 << 2),  //! \deprecated Use output messages instead
+    pe_print_printf         = (1 << 3),  //! \deprecated Use output messages instead
```

Comment:
External code can't use output messages, we just have to deprecate these with no replacement. Presumably anyone who wants to display resource info will be able to use high-level functions from libpacemaker (once they exist). In practice I don't think any external code ever used this.

Reply:
Ah right.  You'd think I would remember that, considering how much I enjoy formatted output being private API.  Fixed.